prerendering.bs

```diff
@@ -974,7 +1000,11 @@ Modify the <a spec=HTML>download the hyperlink</a> algorithm to ensure that down
 
 <h3 id="delay-async-apis">Delaying async API results</h3>
 
-Many specifications need to be patched so that, if a given algorithm invoked in a [=prerendering navigable=], most of its work is deferred until the navigable's [=navigable/top-level traversable=] is [=prerendering traversable/activated=]. This is tricky to do uniformly, as many of these specifications do not have great hygeine around <a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loop-for-spec-authors">using the event loop</a>. Nevertheless, the following sections give our best attempt.
+Many specifications need to be patched so that, if a given algorithm invoked in a [=prerendering navigable=], most of its work is deferred until the navigable's [=navigable/top-level traversable=] is [=prerendering traversable/activated=].
+
+<p class="note">Note that if the [=navigable/scripting mode=] is "`paused-until-activation`", the scripts that would invoke these algorithms are not executed in the first place until activation.</p>
```

Comment:
nit: I feel this note should be better placed after Line 1007, because I was confused as if "This is tricky to do uniformly, ... hygeine around using the event loop" would refer to the paused-until-activation.
(The document structure `<p class=note>` should help disambiguate this, but this confused me because the same thing also applies to paused-until-activation in some sense)